extension/webpack.dev.js

```diff
@@ -0,0 +1,7 @@
+const { merge } = require('webpack-merge');
+const config = require('./webpack.config.js');
+
+module.exports = merge(config, {
+    mode: 'development',
+    devtool: 'inline-source-map',
+});
```

Comment:
```suggestion
});

```
nit: add a new line at EOF